supabase/functions/_backend/public/channel.ts

```diff
@@ -146,8 +146,8 @@ export async function post(c: Context, body: ChannelSet, apikey: Database['publi
     app_id: body.app_id,
     name: body.channel,
     ...(body.public == null ? {} : { public: body.public }),
-    ...(body.disableAutoUpdateUnderNative == null ? {} : { disableAutoUpdateUnderNative: body.disableAutoUpdateUnderNative }),
```

Comment:
the object returned shouldn't change, otherwise the user will have they process broken

Reply:
I don't think I follow. It returns a [BRES/empty response](https://github.com/Cap-go/capgo/blob/0052d865610e243991fd14ba45396917f9c7a5e5/supabase/functions/_backend/public/channel.ts#L209) as this is a POST request. It still accepts the same body parameters as before.